<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddOperator</class>
 <widget class="QWidget" name="AddOperator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="minimumSize">
          <size>
           <width>380</width>
           <height>550</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>380</width>
           <height>550</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>378</width>
            <height>548</height>
           </rect>
          </property>
          <widget class="QScrollArea" name="scrollArea">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>10</y>
             <width>300</width>
             <height>260</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>260</height>
            </size>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>298</width>
              <height>258</height>
             </rect>
            </property>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>10</y>
               <width>121</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Aharoni</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>添加记账员</string>
             </property>
            </widget>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>50</y>
               <width>281</width>
               <height>28</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Andalus</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>用户名</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_jz_name">
                <property name="echoMode">
                 <enum>QLineEdit::Normal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="layoutWidget_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>90</y>
               <width>281</width>
               <height>28</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Andalus</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>密码</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_jz_mm">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="layoutWidget_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>130</y>
               <width>281</width>
               <height>28</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Andalus</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>确认密码</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_jz_qr">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="layoutWidget_4">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>170</y>
               <width>281</width>
               <height>28</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Andalus</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>所属主管</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_jz_chargeNAme">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QPushButton" name="pushButton">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>210</y>
               <width>81</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>添加记账员</string>
             </property>
            </widget>
           </widget>
          </widget>
          <widget class="QScrollArea" name="scrollArea_2">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>300</y>
             <width>300</width>
             <height>230</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>230</height>
            </size>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>298</width>
              <height>228</height>
             </rect>
            </property>
            <widget class="QLabel" name="label_6">
             <property name="geometry">
              <rect>
               <x>110</x>
               <y>10</y>
               <width>121</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Aharoni</family>
               <pointsize>14</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>添加主管</string>
             </property>
            </widget>
            <widget class="QWidget" name="layoutWidget_5">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>50</y>
               <width>281</width>
               <height>28</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Andalus</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>用户名</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_zg_name">
                <property name="echoMode">
                 <enum>QLineEdit::Normal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="layoutWidget_6">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>90</y>
               <width>281</width>
               <height>28</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Andalus</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>密码</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_zg_mm">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="layoutWidget_7">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>130</y>
               <width>281</width>
               <height>28</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Andalus</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>确认密码</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_zg_qr">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QPushButton" name="pushButton_2">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>170</y>
               <width>81</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>添加主管</string>
             </property>
            </widget>
           </widget>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AddOperator</receiver>
   <slot>AddJZY()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>AddOperator</receiver>
   <slot>AddZG()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>568</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddJZY()</slot>
  <slot>AddZG()</slot>
 </slots>
</ui>
